frontend/src/app/main/ui/workspace/sidebar/options/shapes/frame.cljs

```diff
@@ -47,7 +47,8 @@
                         :shape shape}]
      [:& component-menu {:ids comp-ids
                          :values comp-values
-                         :shape-name (:name shape)}]
+                         :shape-name (:name shape)
```

Comment:
If you send the whole shape, the name is not necessary as a parameter.

Reply:
It;s true, fixed